react-whib/src/components/Sidebar.js

```diff
@@ -24,7 +37,13 @@ function Sidebar(){
             <div id='sidebar-menu' className={isActive ? 'menu-open' : null}>
                 <NavLink className='navlink' title='Home' to="/"><img src={homeIcon} /></NavLink>
                 <NavLink className='navlink' title='Map' to="/map"><img src={mapIcon} /></NavLink>
-                <NavLink className='navlink' title='Profile' to="/profile"><img src={profileIcon} /></NavLink>
+                {token != 'null' ? 
```

Comment:
same as before, if you just remove token entirely then it should work with token != null.

At least i think so